dashboard/app/controllers/activities_controller.rb

```diff
@@ -165,26 +164,7 @@ def milestone
     authorize! :create, UserLevel
 
     test_result = params[:testResult].to_i
-    solved = (params[:result] == 'true')
-
-    lines = params[:lines].to_i
```

Comment:
Are these query parameters no longer used? Should we pull this thread further and stop submitting them too?

Reply:
Both are still referenced here:
https://github.com/code-dot-org/code-dot-org/blob/0da970a125b3ab623bde61d779ff716d187ee0a1/dashboard/app/controllers/activities_controller.rb#L261-L262